metadata-ingestion/src/datahub/ingestion/source/tableau.py

```diff
@@ -286,16 +305,57 @@ class TableauConfig(
         description="Ingest a URL to render an embedded Preview of assets within Tableau.",
     )
 
+    extract_project_hierarchy: bool = Field(
```

Comment:
this is a breaking change right?

Reply:
No, as it is saying whether to extract project hierarchy for project provided in configuration.
Is there any reason you feel it is breaking change ?